tutorials/db-migration-spanner-handle-increasing-pks/index.md

```diff
@@ -0,0 +1,363 @@
+*   Striim License, which includes a trial period through the [Cloud Marketplace](https://console.cloud.google.com/marketplace/details/striim/striim)
+
+Use the [pricing calculator](https://cloud.google.com/products/calculator) to generate a cost estimate based on your projected usage.   
+
+## Before you begin
+
+For this tutorial, you need a Google Cloud [project](https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy#projects). You can create a
+new one, or you can select a project that you already created.
+
+1.  [Select or create a Google Cloud project.](https://console.cloud.google.com/projectselector2/home/dashboard)
+2.  [Enable billing for your project.](https://support.google.com/cloud/answer/6293499#enable-billing)
+3.  [Enable API](https://console.cloud.google.com/flows/enableapi?apiid=compute.googleapis.com,spanner.googleapis.com) for Compute Engine and Spanner.
+
+When you finish this tutorial, you can avoid continued billing by deleting the resources that you created. For details, see the "Cleaning up" 
+section at the end of this tutorial.
+
+## Setup Source Database (MySQL)
+1. Create Cloud SQL Instance, in a cloud shell.
+
+        gcloud sql instances create mysql-57  \
+            --database-version=MYSQL_5_7 \
+            --tier=db-n1-standard-1  \
+            --region=us-central1 \
+            --root-password=password123
+
+2. Enable binary logging (for CDC).
+
+        gcloud sql instances patch mysql-57 --backup-start-time 00:00
+        gcloud sql instances patch mysql-57 --enable-bin-log    
+
+3. Connect to mysql instance. Type in password *password123* when prompted.
+
+        gcloud sql connect mysql-57 --user=root
+
+4. Create database and tables.
+
+        create database employeedb;
+        use employeedb;
+        CREATE TABLE `signin_log` (
+          `id` int not null AUTO_INCREMENT,
+          `employee_email` varchar(200) DEFAULT NULL,
+          `details` varchar(500) DEFAULT NULL,
+          PRIMARY KEY (`id`)
+        );
+    
+5. Insert sample data rows.
+
+        insert into signin_log (employee_email, details) values ('test-1@email.com', 'ip address a.b.c.d');
+        insert into signin_log (employee_email, details) values ('test-2@email.com', 'new ip address 1');
+
+## Setup Target Database (Cloud Spanner)
+1. Create a cloud spanner instance
+
+        gcloud spanner instances create spanner-tgt \
+            --config=regional-us-central1 \
+            --nodes=1 \
+            --description=spanner-tgt
+
+2. Create database and empty table.
+
+        gcloud spanner databases create employeedb \
+        --instance=spanner-tgt \
+        --ddl='CREATE TABLE signin_log (id STRING(36), employee_email STRING(200), details STRING(500)) PRIMARY KEY (id)'
+        
+**NOTE** You might have noticed that employee_id has been changed to STRING in the above example. As discussed previously, this is done so that application can generate UUID keys post migration.
```

Comment:
```suggestion
**NOTE** The employee_id column has been changed to STRING in the above example. As mentioned previously, this is done so that application can generate UUID keys post migration.
```